app/_hub/kong-inc/jq/index.md

```diff
@@ -42,7 +42,6 @@ params:
   route_id: true
   consumer_id: true
   yaml_examples: false
-  k8s_examples: false
```

Comment:
I don't either. Up to the plugin devs to add it if they want, but I think this is fine: if it gets examples enabled in general, it will have K8S examples.